client/src/components/FriendSuggestions/FriendSuggestionItem/FriendSuggestionItem.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+import { Grid, IconButton, Paper, Tooltip, Typography } from '@material-ui/core'
+import PanToolOutlinedIcon from '@material-ui/icons/PanToolOutlined'
+import CallMadeOutlinedIcon from '@material-ui/icons/CallMadeOutlined'
+
+import useStyles from './friendSuggestionItemStyles'
+
+import { sendFriendRequest } from '../../../actions/friends'
+import { getAvatarLink } from '../../../utils/helpers/imageLinkHelpers'
+
+const COMMON_F_AVATARS_TO_SHOW = 4
+
+const FriendSuggestions = ({ person, commonFriends }) => {
+  const classes = useStyles({ avatar: getAvatarLink(person.avatar) })
+  const [ requestSent, setRequestSent ] = useState(false)
+
+  const createFriendRequest = responderId => {
+    sendFriendRequest(responderId)
+      .then(() => setRequestSent(true))
+  }
+
+  const commonFriendAvatars = commonFriends
+    .filter(cf => cf.avatar !== null)
```

Comment:
For dev purposes only?

Reply:
Actually, not. Didn't want to show friends without avatars